docs/documentation/overview/responsiveness.html

```diff
@@ -27,81 +38,71 @@ <h2 class="subtitle">Bulma is a <strong>mobile-first</strong> framework</h2>
     </div>
 
     {% include anchor.html name="Breakpoints" %}
+    
+    {% assign variables_file_url = "/blob/master/sass/utilities/initial-variables.sass#L46,L57" | prepend: site.github %}
+    {% assign mixins_file_url = "/blob/master/sass/utilities/mixins.sass#L182,L226" | prepend: site.github %}
 
     <div class="content">
-      <p>Bulma has 5 breakpoints:</p>
+      <p>Bulma has <a href="{{ variables_file_url }}" target="_blank">5 breakpoints</a>:</p>
       <ul>
-        <li><code>mobile</code>: up to <code>768px</code></li>
-        <li><code>tablet</code>: from <code>769px</code></li>
-        <li><code>desktop</code>: from <code>1008px</code></li>
-        <li><code>widescreen</code>: from <code>1200px</code></li>
-        <li><span class="tag is-success">New!</span> <code>fullhd</code>: from <code>1392px</code></li>
+        {% for breakpoint_hash in site.data.breakpoints %}
+          {% assign breakpoint = breakpoint_hash[1] %}
+          <li>{% if breakpoint.id == 'fullhd' %}<span class="tag is-success">New!</span> {% endif %}<code>{{ breakpoint.id }}</code>: {% if breakpoint.id == 'Mobile' %}up to <code>{{ breakpoint.to }}px</code>{% else %}from <code>{{ breakpoint.from }}px</code>{% endif %}</li>
```

Comment:
Sorry about that, I just noticed that I used `Mobile` instead of `mobile`;

Should be `{% if breakpoint.id == 'mobile' %}` instead of `{% if breakpoint.id == 'Mobile' %}`.

Can you update it or do you want me to create a new PR?